### PR TITLE
CHI-1019 terraform acct dirs

### PR DESCRIPTION
**Pull request by stephenhand**
Once this is merged, I will migrate Aarambh Prod & SafeSpot Staging accounts (both of their `terraform plan`s look good) and raise a follow up PR that can delete all the 'moved' directives, as there will be no accounts left with the old terraform config.

Also, merging this will mean the setting up of the 3 new helplines can commence

## Description

* Moves the terraform-poc account Terraform config into its own directory
* Update docs

TODO

* Move Aarambh Production & Safespot Staging into their own directories
* Include instructions about migrating states

### Checklist
- [ ] Corresponding issue has been opened
- [ ] New tests added
- [ ] Strings are localized

### Related Issues
Fixes CHI-1019

### Verification steps

* Migrate an existing helpline from the single main.tf (instructions to come)
* Create a new helpline, following the revised instructions in the readme

### CODE REVIEW

File: scripts/src/terraformSupplemental/importDefaultTwilioResourcesToTerraform.ts [hunk 1]
```diff
@@ -6,8 +6,8 @@ import { logError, logSuccess, logWarning } from '../helpers/log';
 
 config();
 
-const TERRAFORM_WORKING_DIRECTORY = '../twilio-iac/aselo-terraform';
 
+const terraformWorkingDirectory = `../twilio-iac/${process.argv[2]}`;
```
Comment (stephenhand) [inline scripts/src/terraformSupplemental/importDefaultTwilioResourcesToTerraform.ts:8]: Changes in this script haven't been tested, might need some fixing when we try them on a new account with the new structure

File: twilio-iac/terraform-modules/chatbots/default/main.tf [hunk 2]
```diff
@@ -359,55 +359,45 @@ resource "twilio_autopilot_assistants_field_types_field_values_v1" "unknown_age_
...
-  for_each = toset(["Boy", "Girl", "Unknown", "Non-Binary"])
-  assistant_sid = twilio_autopilot_assistants_v1.pre_survey.sid
-  field_type_sid = twilio_autopilot_assistants_field_types_v1.gender.sid
-  value = each.key
-  language = "en-US"
+module "safespot_gender_field" {
+  count = var.gender_field_type == "safespot" ? 1 : 0
+  source = "../terraform-modules/gender-field/safespot"
+  bot_sid = twilio_autopilot_assistants_v1.pre_survey.sid
 }
```
Comment (murilovmachado) [inline twilio-iac/terraform-modules/chatbots/default/main.tf:372]: So we will have one resource like this per helpline that is not `default`, right?

File: twilio-iac/terraform-poc-account/main.tf [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    }
+  }
+
+  backend "s3" {
+    bucket         = "tl-terraform-state-twilio-terraform-poc"
+    key            = "twilio/terraform.tfstate"
+    dynamodb_table = "twilio-terraform-terraform-poc-locks"
+    encrypt        = true
+  }
+}
```
Comment (murilovmachado) [inline twilio-iac/terraform-poc-account/main.tf:15]: Do we still need this `terraform-poc-account` folder?

**Review by murilovmachado — APPROVED** [review]
LGTM!

File: twilio-iac/terraform-poc-account/main.tf (hunk 3 above)
Comment (stephenhand) [reply twilio-iac/terraform-poc-account/main.tf:15]: Not 'need', but I find it a useful testbed for developing terraform changes safely. When we have Aselo Dev under terraform (if we do that) we could probably get rid of this but I wouldn't mind keeping it around for now

File: twilio-iac/terraform-modules/chatbots/default/main.tf (hunk 2 above)
Comment (stephenhand) [reply twilio-iac/terraform-modules/chatbots/default/main.tf:372]: Exactly, each helpline that diverges with gender choices can implement their own module - and if a helpline doesn't gather gender info with the bot they could omit it entirely

**Pull request merged**